CONTRIBUTING.md

```diff
@@ -65,5 +65,5 @@ Integration tests are declared in [`./tests`](./tests) as subdirectories imitati
 
 Pinned dependencies are [regularly updated automatically](./.github/workflows/update.yml).
 
-Releases are [automatically created](./.github/workflows/release.yml) when the `version` field in [`Cargo.toml`](./Cargo.toml)
+Releases are [automatically created](./.github/scripts/release.sh) when the `version` field in [`Cargo.toml`](./Cargo.toml)
```

Comment:
```suggestion
Releases are [automatically created](./.github/workflows/after-release.yml) when the `version` field in [`Cargo.toml`](./Cargo.toml)
```

Reply:
Oh I see that you intentionally changed it to `scripts/release.sh`. I think pointing to the workflow is better, since that's what causes the script to run automatically (the script by itself could also be run manually).

Reply:
I thought `main.yml` caused release.sh to run?

Reply:
Oh damn you're right! I take it back, `release.sh` seems much better to point to now :laughing:

Reply:
Lol I clicked auto-merge and it merged it before we could fix it back.

Reply:
Opened https://github.com/NixOS/nixpkgs-check-by-name/pull/57 to fix this